ml-proto/spec/kernel.ml

```diff
@@ -50,16 +50,16 @@ struct
            | ReinterpretInt
 end
 
-module Int32Op = IntOp ()
-module Int64Op = IntOp ()
-module Float32Op = FloatOp ()
-module Float64Op = FloatOp ()
+module I32Op = IntOp ()
+module I64Op = IntOp ()
+module F32Op = FloatOp ()
+module F64Op = FloatOp ()
```

Comment:
Well, it's a module defining the set of i32 operators. It's created from a generic module defining integer operators. But there is nothing new here, I just renamed them to match the Wasm type names better.